ui/src/components/SchemaForm/Widgets/CheckboxWidget.tsx

```
@@ -0,0 +1,93 @@
+    "DescriptionFieldTemplate",
+    T,
+    S,
+    F
+  >("DescriptionFieldTemplate", registry, options);
+  // Because an unchecked checkbox will cause html5 validation to fail, only add
+  // the "required" attribute if the field value must be "true", due to the
+  // "const" or "enum" keywords
+  const required = schemaRequiresTrueValue<S>(schema);
+
+  const _onChange = ({ target }: FocusEvent<HTMLButtonElement>) =>
+    onChange(id, target && target.value);
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)